tools/c7n_azure/tests_azure/tests_resources/test_advisor.py

```diff
@@ -0,0 +1,26 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from ..azure_common import BaseTest
+
+
+class AdvisorRecommendationsTest(BaseTest):
+    def test_azure_advisor_recommendations_schema_validate(self):
+        p = self.load_policy({
+            'name': 'test-azure-advisor-recommendations',
+            'resource': 'azure.advisor-recommendations'
+        }, validate=True)
+        self.assertTrue(p)
+
+    def test_find_by_name(self):
+        p = self.load_policy({
+            'name': 'test-azure-advisor-recommendations',
+            'resource': 'azure.advisor-recommendations',
+            'filters': [
+                {'type': 'value',
+                 'key': 'name',
+                 'op': 'glob',
+                 'value_type': 'normalize',
+                 'value': 'ccadvisorrecommendations*'}],
```

Comment:
Two feedback items about the test that is failing now:

1. Given that recommendation don't really have any names, I'd suggest removing the filter entirely and updating an assert with `len(resources) > 0`
2. Mark this test with `@pytest.mark.skiplive` because there is no reliable way of creating a recommendation in test accounts